src/main/java/mate/academy/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,25 @@
 package mate.academy.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import mate.academy.bookshop.validator.Isbn;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
to validate String use NotBlank

```suggestion
    @NotBlank
```